python/ray/serve/multiplex.py

```diff
@@ -94,3 +147,22 @@ async def unload_model(self) -> None:
             else:
                 await sync_to_async(model.__del__)()
             setattr(model, "__del__", lambda _: None)
+
+    async def _push_model_ids(self):
```

Comment:
Is it possible to piggy-back this onto the existing messages that record the autoscaling stats?

Reply:
synced offline, keep as it is right now.